opm/material/fluidsystems/blackoilpvt/Co2GasPvt.hpp

```diff
@@ -148,19 +164,26 @@ class Co2GasPvt
     Evaluation inverseFormationVolumeFactor(unsigned regionIdx,
                                             const Evaluation& temperature,
                                             const Evaluation& pressure,
-                                            const Evaluation& /*Rv*/,
-                                            const Evaluation& /*Rvw*/) const
-    { return saturatedInverseFormationVolumeFactor(regionIdx, temperature, pressure); }
+                                            const Evaluation& rv,
+                                            const Evaluation& rvw) const
+    {
+        // assume ideal mixture
+        const Evaluation xBrine = convertRvwToXgW_(Opm::max(rvw,rv),regionIdx);
```

Comment:
We're already in namespace Opm here, so you should either drop the namespace qualification (`max(rvw,rv)`) or use a globally anchored namespace qualification (`::Opm::max(rvw,rv)`).